bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuntimeRule.java

```diff
@@ -79,7 +93,7 @@ public Module getModule(String moduleId) {
     }
 
     private static List<Condition> getRuntimeConditionsCopy(List<Condition> conditions) {
-        List<Condition> res = new ArrayList<Condition>();
+        List<Condition> res = new ArrayList<Condition>(11);
```

Comment:
Why 11? Do such magic numbers really help us in any way? I doubt that the potential effect is anything but insignificant...

Reply:
Agree, will remove it